lib/datadog/ci/transport/http.rb

```diff
@@ -149,6 +149,10 @@ def code
             nil
           end
 
+          def response_size
+            0
```

Comment:
unrelated change, I noticed that library started to fail if certain backend API is not available because for ErrorResponse there is no http_response